lib/chart/internal/figure.ex

```diff
@@ -1,18 +1,17 @@
 defmodule Chart.Internal.Figure do
   @moduledoc false
 
-  @fig_viewbox {800, 600}
+  import Chart.Internal.Guards, only: [is_positive_numbers: 2]
 
   def new() do
-    %{viewbox: @fig_viewbox}
+    %{viewbox: {800, 600}}
   end
 
   def add(settings) when is_map(settings) do
     Map.put(settings, :figure, new())
   end
 
-  def put_viewbox(settings, {width, height} = viewbox)
-      when is_number(width) and is_number(height) and 0 < width and 0 < height do
+  def put_viewbox(settings, {width, height} = viewbox) when is_positive_numbers(width, height) do
```

Comment:
Toto neni uplne dobra zkratka, kdyz budes mit nekde tri parametery `x, y, z`, tak budes delat guard `is_positive_numbers/3`? Rozhodne je lepsi mit jen `is_positive_number`.

Reply:
Oki opravim to. Jsem myslel, ze u guards mi to nebude fungovat kdyz budu mit vicero stejnych nazvu s jinymi arity

Reply:
spis je normalni pouzit `is_postive_number` vicerkrat, pr:

```elixir
def f(x, y) when is_postive_number(x) and is_postive_number(y) do...
```

Reply:
oki opravim to